src/arkouda_server.chpl

```diff
@@ -67,7 +67,7 @@ proc main() {
             else {
                 writeln("reqMsg: ", reqMsg);
             }
-            writeln(">>> ",cmd," started at ",t1,"sec");
+            writeln(">>> %s started at %.17r sec".format(cmd, t1));
```

Comment:
The format specifier `%.17r` will work but says "give me 17 significant digits". It might be easier to use the [generic number formats](https://chapel-lang.org/docs/master/modules/standard/IO/FormattedIO.html?highlight=print%20specifier#generic-numeric-conversions) if you only want so many digits after the decimal point.

For example, using format specifier `%{.###}` gives you three digits after the point. [See example in TIO.](https://tio.run/##hcw9CoNAFATg3lMMSsA0K5LCQlLlCrnAos/1ga6y@4xFyNk3/jQhIVjOMPNVrR6pC2HyhDv3VEbRQzvYYcZ1K5QhuU3OkZU1pucymh0LdTZdRh8plpZQ7UvIMgV7nFReuFg1g@u1bIdzCSDLkBfwbBo2HjKI7o6d@hfanZoNi4duhBzW9ziwlQPwqZIkeX2LC4jLHzCENw)

For info, the `Time` module calls `chpl_now_time()` -> `gettimeofday` which has up-to-microsecond resolution.

Reply:
I wanted to allow for all the precision of floating point number from the IEEE 754 spec which states that 17 digits of precision captures all the precision of a 64 bit floating point number.

Reply:
Ah fair enough. Good point.